### ISSUE TITLE
App won't open on Windows sometimes

### ISSUE THREAD

**Issue by sguergachi** [careteditor/caret#705]
Sometimes I find when I try launching the Caret app on Windows 10 (latest) It doesn't open

Looking at the Task Manager I see this:
![image](https://user-images.githubusercontent.com/615386/38054198-43620198-32a4-11e8-8921-e265e4b0ea95.png)

It seems every time I click the shortcut (links to C:\Users\magis\AppData\Local\caret-beta\caret-beta.exe, starts in: C:\Users\magis\AppData\Local\caret-beta\app-4.0.0-rc2) a new process is created but the app doesn't start.

The workaround is to close all the processes and launch again.

**Comment by astoilkov** [comment]
Hi Sammy,

Has this happened multiple times? How often does it happen?

Thanks in advance.

**Comment by sguergachi** [comment]
Yes this has happened multiple times, though I’m not sure how to reproduce it.
